src/heartbeat-loop/heartbeat-loop.ts

```diff
@@ -0,0 +1,56 @@
+import { createSha256Hash, serializePlainObject } from '@api3/commons';
+import { go } from '@api3/promise-utils';
+import { ethers } from 'ethers';
+
+import { loadRawConfig } from '../config';
+import { HEARTBEAT_LOG_MESSAGE } from '../constants';
+import { logger } from '../logger';
+import { getState } from '../state';
+
+import { heartbeatLogger } from './logger';
+
+export const startHeartbeatLoop = () => {
+  logger.info('Initiating heartbeat loop.');
+
+  setInterval(async () => {
+    const goLogHeartbeat = await go(logHeartbeat);
+    if (!goLogHeartbeat.success) logger.error('Failed to log heartbeat.', goLogHeartbeat.error);
+  }, 1000 * 60); // Frequency is hardcoded to 1 minute.
+};
+
+export interface HeartbeatPayload {
+  currentTimestamp: string;
+  configHash: string;
+  signature: string;
+}
+
+export const logHeartbeat = async () => {
+  logger.debug('Creating heartbeat log.');
+
+  const rawConfig = loadRawConfig(); // We want to log the raw config, not the one with interpolated secrets.
+  const configHash = createSha256Hash(JSON.stringify(rawConfig));
```

Comment:
```suggestion
  const configHash = createSha256Hash(serializePlainObject(rawConfig));
```

As I remember, this was thing requires fix in `airnode-feed` right?

Reply:
Yeah, thanks for that. I'd probably miss this.